status/status_test.go

```diff
@@ -348,11 +349,16 @@ func str(s *Status) string {
 
 // mustMarshalAny converts a protobuf message to an any.
 func mustMarshalAny(msg proto.Message) *apb.Any {
-	any, err := anypb.New(msg)
+	protoMessage, ok := msg.(protoreflect.ProtoMessage)
```

Comment:
Should i check message if its coming from protoreflect.ProtoMessage package, anypb.New() function expects a message type that implements the proto.Message interface from the google.golang.org/protobuf/proto

However, the msg argument that we are passing to this function is still of type github.com/golang/protobuf/proto.Message, which does not implement the proto.Message interface.

@easwars